packages/igx-templates/igx-ts/projects/side-nav-auth/files/package.json

```
@@ -33,9 +33,9 @@
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "0.900.0-rc.7",
```

The `@angular-devkit` devDep should also be non-strict
```suggestion
    "@angular-devkit/build-angular": "~0.900.0-rc.0",
```